lib/nostrum/api.ex

```diff
@@ -2938,4 +2960,44 @@ defmodule Nostrum.Api do
 
     {:ok, convert}
   end
+
+  def prepare_allowed_mentions(options) do
```

Comment:
Might as well make this `defp` as well IMO

Reply:
Sorry, was testing the function and forgot to make it private again